docs/table-of-contents.json

```diff
@@ -179,11 +195,133 @@
             },
             {
               "name": "Object3d",
-              "entry": "core/object-3d"
+              "entry": "docs/api-reference/core/object-3d"
             },
             {
               "name": "Group",
-              "entry": "docs/core/group"
+              "entry": "docs/api-reference/core/group"
+            },
+            {
+              "name": "Transform",
+              "entry": "docs/api-reference/core/transform"
+            }
+          ]
+        },
+        {
+          "title": "addons",
```

Comment:
For the following 4 entries, the corresponding folders were not taken into account in the TOC